### DISCUSSION TITLE
CTD with some liveries

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6951] [General]
Hi dev team,

i recognized some CTD with 3rd party aircraft textures. Mostly elderly ones. What might be the trigger for these CTD´s ?

greets CptB

**Comment by NathanInnes** [comment]
Is this issue still persistant?

**Reply by CptBayer** [reply]
Hi,

wason holidays and hadnoflight for alonger time .... I willcheck it within the next days...

greets Klaus
